protos/sources/vega/vega.proto

```diff
@@ -872,6 +878,18 @@ enum TransferType {
   TRANSFER_TYPE_HOLDING_RELEASE = 28;
   // Insurance pool fraction transfer from parent to successor market.
   TRANSFER_TYPE_SUCCESSOR_INSURANCE_FRACTION = 29;
+  // Allocate liquidity fee among liquidity providers to their network controlled liquidity fee account.
+  TRANSFER_TYPE_LIQUIDITY_FEE_ALLOCATE = 30;
+  // Distributes net fee from liquidity provider's fee account to their general account.
+  TRANSFER_TYPE_LIQUIDITY_FEE_NET_DISTRIBUTE = 31;
+  // Applies SLA penalty by moving funds from party bond account to market insurance pool.
+  TRANSFER_TYPE_SLA_PENALTY_BOND_APPLY = 32;
+  // Applies SLA penalty by moving funds from the liquidity provider's fee account to market insurance pool.
+  TRANSFER_TYPE_SLA_PENALTY_LP_FEE_APPLY = 33;
+  // Collects unpaid rewards from the liquidity provider's fee account to the market bonus distribution account.
+  TRANSFER_TYPE_LIQUIDITY_FEE_UNPAID_COLLECT = 34;
```

Comment:
I think the name for this could be clearer, I'm not sure if unpaid_collect explains what's going on here.